public/docs/_examples/cb-third-party-lib/hero-profile/hero-profile.component.ts

```diff
@@ -0,0 +1,14 @@
+// #docregion
+import { Component, Input } from '@angular/core';
+
+import { Hero } from './hero';
+
+@Component({
+  moduleId: module.id,
+  selector: 'hero-profile',
+  templateUrl: 'hero-profile.component.html',
+  styleUrls: ['hero-profile.component.css']
```

Comment:
https://github.com/angular/angular.io/pull/3071 changed all paths to be relative, these should be as well.